package.json

```diff
@@ -48,13 +48,13 @@
     "fast-json-stringify": "."
   },
   "dependencies": {
-    "@fastify/deepmerge": "^1.0.0",
     "ajv": "^8.10.0",
     "ajv-formats": "^2.1.1",
     "fast-deep-equal": "^3.1.3",
     "fast-uri": "^2.1.0",
     "rfdc": "^1.2.0",
-    "json-schema-ref-resolver": "^1.0.1"
+    "json-schema-ref-resolver": "^1.0.1",
+    "json-schema-merge-allof": "^0.8.1"
```

Comment:
The use of https://www.npmjs.com/package/json-schema-merge-allof does not inspire me a lot of faith as it seems an unmaintained module. Is there an alternative? Can we fork or embed its code in here?

Reply:
Unfortunatelly this is the best solution I could find. We can write our own solution, but I don't have time for this right now. Anyway it should be better that our currect merge logic.